services/network/wpa_supplicant.py

```diff
@@ -0,0 +1,868 @@
+import os
+import socket
+import time
+
+class WPASupplicant:
+    BUFFER_SIZE = 4096
+    target = ('0.0.0.0', 6664)
+    sock = None
+    verbose = True
+
+    def __del__(self):
+        if self.sock:
+            self.sock.close()
+
+    def run(self, target=target):
+        """Does the connection and setup variables
```

Comment:
document "target", it seems it can be either udp or a unix socket, but it is not clear